src/OOVPADatabase/D3D8LTCG/4039.inl

```diff
@@ -85,6 +85,33 @@ OOVPA_SIG_MATCH(
     // Do not use any offsets after 0x1B
 );
 
+// test case: NBA2K2
+//  ******************************************************************
+//  * D3D::CDevice::KickOff
+//  ******************************************************************
+OOVPA_SIG_HEADER_XREF(CDevice_KickOff_0_edx,
+                      4039,
+                      XRefOne)
+OOVPA_SIG_MATCH(
+    // mov eax, XREF_D3D_g_pDevice
```

Comment:
No XREF_ prefix in comments. And there are far too many XREF_ prefix used in comments across all changed files which I will not repeat for each one.

I am willing to reformat these comments to use our current format regardless older signatures aren't following the format we expected. In a separate commit of course.